services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -7824,6 +7826,38 @@ public Optional<PushStatusStoreWriter> getPushStatusStoreWriter() {
     return pushStatusStoreWriter;
   }
 
+  @Override
+  public void sendHeartbeatToSystemStore(String clusterName, String storeName, long heartbeatTimeStamp) {
```

Comment:
Do we need to check whether the push status store is enabled or not in these functions?

Reply:
Let me leave a comment there indicating that PushStatusStore is fully rolled out. We should clean up these configs and move optional to required arguments.